browser/react.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  extends: ['plugin:react/recommended', 'plugin:react-hooks/recommended'],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    sourceType: 'module',
+  },
+  env: {
+    browser: true,
+  },
+  globals: {
+    /**
+     * We need to globalize `analytics` because of the way Segment's library loads. It redefines
+     * `window.analytics` after loading, which, when `analytics` is not globalized, causes issues
+     * because the `analytics` rollup catches may not be the `window.analytics` we want.
+     */
+    analytics: true,
+  },
```

Comment:
Are these settings React-specific? Should they live in the other file?

Reply:
Good point. I think it makes sense for them to be combined, but I'll rename the file so it's not react-specific,